<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>H2Team</class>
 <widget class="QMainWindow" name="H2Team">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1368</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>70</y>
      <width>61</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="qa">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Quay ảnh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pt_90">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>180</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Xoay ảnh 90 độ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pt_reset">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>530</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reset ảnh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>650</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ảnh gốc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>640</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ảnh biến đổi</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_lb1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>501</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_lb2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>501</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pt_axam">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>290</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Chuyển thành ảnh xám</string>
    </property>
   </widget>
   <widget class="QSlider" name="sl_tll">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>460</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="pageStep">
     <number>15</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>410</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tỉ lệ lọc</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1368</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_File"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_nca">
    <property name="title">
     <string>Nâng cấp ảnh</string>
    </property>
    <widget class="QMenu" name="menu_ptda">
     <property name="title">
      <string>Phép toán điểm ảnh</string>
     </property>
     <addaction name="action_Daoanh"/>
     <addaction name="action_Catng"/>
     <addaction name="action_Gamma"/>
    </widget>
    <widget class="QMenu" name="menu_Kthistogram">
     <property name="title">
      <string>Kỹ thuật histogram</string>
     </property>
     <addaction name="action_cbhist"/>
     <addaction name="action_histanh"/>
    </widget>
    <addaction name="menu_ptda"/>
    <addaction name="menu_Kthistogram"/>
   </widget>
   <widget class="QMenu" name="menu_la">
    <property name="title">
     <string>Lọc ảnh</string>
    </property>
    <widget class="QMenu" name="menu_ma">
     <property name="title">
      <string>Lọc mịn ảnh</string>
     </property>
     <addaction name="action_locgau"/>
     <addaction name="action_Tv"/>
    </widget>
    <widget class="QMenu" name="menu_tba">
     <property name="title">
      <string>Lọc tìm biên ảnh</string>
     </property>
     <addaction name="action_loclap"/>
    </widget>
    <addaction name="menu_ma"/>
    <addaction name="menu_tba"/>
   </widget>
   <widget class="QMenu" name="menu_nhieu">
    <property name="title">
     <string>Nhiễu ảnh </string>
    </property>
    <addaction name="action_nhieugau"/>
    <addaction name="action_nhieumt"/>
   </widget>
   <widget class="QMenu" name="menu_cd">
    <property name="title">
     <string>Cài đặt</string>
    </property>
    <widget class="QMenu" name="menu_doimau">
     <property name="title">
      <string>Đổi màu</string>
     </property>
     <addaction name="action_den"/>
     <addaction name="action_trang"/>
     <addaction name="action_xanh"/>
    </widget>
    <addaction name="action_pt"/>
    <addaction name="action_tn"/>
    <addaction name="menu_doimau"/>
   </widget>
   <widget class="QMenu" name="menu_gt">
    <property name="title">
     <string>Giới thiệu</string>
    </property>
    <addaction name="action_ud"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_nca"/>
   <addaction name="menu_la"/>
   <addaction name="menu_nhieu"/>
   <addaction name="menu_cd"/>
   <addaction name="menu_gt"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit </string>
   </property>
  </action>
  <action name="action_gamma">
   <property name="text">
    <string>Biến đổi hàm mũ</string>
   </property>
  </action>
  <action name="action_Daoanh">
   <property name="text">
    <string>Đảo ảnh</string>
   </property>
  </action>
  <action name="action_Catng">
   <property name="text">
    <string>Cắt ngưỡng</string>
   </property>
  </action>
  <action name="action_Gamma">
   <property name="text">
    <string>Biến đổi hàm mũ</string>
   </property>
  </action>
  <action name="action_cbhist">
   <property name="text">
    <string>Cân bằng histogram</string>
   </property>
  </action>
  <action name="action_histanh">
   <property name="text">
    <string>Histogram của ảnh</string>
   </property>
  </action>
  <action name="action_lap">
   <property name="text">
    <string>Lọc Laplace</string>
   </property>
  </action>
  <action name="action_Gau">
   <property name="text">
    <string>Lọc Gaussian</string>
   </property>
  </action>
  <action name="action_tv">
   <property name="text">
    <string>Lọc trung vị</string>
   </property>
  </action>
  <action name="action_nhieugau">
   <property name="text">
    <string>Nhiễu gaussian</string>
   </property>
  </action>
  <action name="action_nhieumt">
   <property name="text">
    <string>Nhiễu muối tiêu</string>
   </property>
  </action>
  <action name="action_pt">
   <property name="text">
    <string>Phóng to</string>
   </property>
  </action>
  <action name="action_tn">
   <property name="text">
    <string>Thu nhỏ</string>
   </property>
  </action>
  <action name="action_ud">
   <property name="text">
    <string>Ứng dụng</string>
   </property>
  </action>
  <action name="action_locgau">
   <property name="text">
    <string>Lọc Gaussian</string>
   </property>
  </action>
  <action name="action_Tv">
   <property name="text">
    <string>Lọc trung vị</string>
   </property>
  </action>
  <action name="action_loclap">
   <property name="text">
    <string>Lọc Laplace</string>
   </property>
  </action>
  <action name="action_den">
   <property name="text">
    <string>Xanh biển</string>
   </property>
  </action>
  <action name="action_trang">
   <property name="text">
    <string>Hồng hườm</string>
   </property>
  </action>
  <action name="action_xanh">
   <property name="text">
    <string>Xanh ngọc bích</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
